src/app/zeko/compile_simple/compile_simple.mli

```diff
@@ -1,5 +1,9 @@
 module Proof : sig
   type t [@@deriving yojson]
+
+  val to_pickles : t -> Pickles.Side_loaded.Proof.t
```

Comment:
You can't have to_pickles as I've explained already.

Reply:
Fake proof is being sent to the testing ledger during tests.

Reply:
Why do you need this for real proof?